Fix template error in c++:

In file included from bmtool-main.cpp:1:
../general/cgetopt.hpp:272:7: error: too many template-parameter-lists
 class COptArg<pair<T,T> > : public COption
       ^~~~~~~~~~~~~~~~~~~

--- general/cgetopt.hpp.orig	2019-10-16 13:05:58.959292445 +0200
+++ general/cgetopt.hpp	2019-10-16 13:06:06.375140664 +0200
@@ -267,7 +267,6 @@
     int m_cnt;
 };
 
-template<>
 template<class T>
 class COptArg<pair<T,T> > : public COption
 {
